internal/utils/csi.go

```diff
@@ -41,6 +41,8 @@ const (
 
 	logsDirVolumeName      = "logs-dir"
 	logRotateDirVolumeName = "log-rotate-dir"
+
+	snapshotMetadatagRPCServicePort = 50051
```

Comment:
define this similar to csi-addons port and add the port to the container spec as well